tools/benchmarking/benchmark_ov.py

```diff
@@ -33,16 +35,23 @@ def benchmark_app(omz_network, skip, batch_size, num_threads, precision, cache_d
             'Downloading {0}'.format(omz_network))  #TODO(brian): make this spit out output easily
         sh.omz_downloader('--cache_dir={0}'.format(cache_dir), name=omz_network, o=model_dir)
         print('Running model optimizer on {0}'.format(omz_network))
+        # add_mo_arg='--disable_nhwc_to_nchw',
         sh.omz_converter(name=omz_network, d=model_dir, o=model_dir, precision=precision)
     else:
         print('Skipping download & conversion')
 
     print('Benchmarking...')
     model_path = os.path.join(model_dir, 'public', omz_network, precision, omz_network + '.xml')
-    ba = sh.Command('benchmark_app')
-    ba = ba.bake(m=model_path, api='sync', t='15', b=batch_size, _long_prefix='-', _long_sep=' ')
+    ba = sh.Command('../.././vendor/openvino/bin/intel64/Release/benchmark_app')
```

Comment:
Maybe we should undo this?

Reply:
I think it's ok to merge this commit. It enables OV benchmark.